.github/workflows/check.yaml

```diff
@@ -0,0 +1,38 @@
+name: Build, linters, and unit tests
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build-linters-unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.x
+
+    - run: |
+        pip3 install --user --upgrade j2cli
+        j2 --version
+
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.19
+
+    - name: Build
+      run: make build
+
+    - name: Linters
+      uses: golangci/golangci-lint-action@v3
```

Comment:
It would be nice to add the linting to the Makefile and then invoke the `make lint` here. Also similar to nmpolicy.

Reply:
will do it on a follow PR, this one adds the `make build` and `make test` (removed the previous lint one)